common/ferrostar/src/routing_adapters/osrm/mod.rs

```diff
@@ -1,18 +1,21 @@
 //! Response parsing for OSRM-compatible JSON (including Stadia Maps, Valhalla, Mapbox, etc.).
 
+pub(crate) mod algorithms;
```

Comment:
Similarly, this is more of a utility module than actual algorithms.

Does this need to be `pub(crate)`? Seems like it's more internal but maybe I missed something.

Reply:
Fixed & renamed to utilities.